discord/commands/options.py

```diff
@@ -41,6 +41,7 @@
     "StageChannel": ChannelType.stage_voice,
     "CategoryChannel": ChannelType.category,
     "Thread": ChannelType.public_thread,
+    "ThreadOption": ChannelType.public_thread,
```

Comment:
I believe `ThreadOption` was meant to be used to chose between public and private threads.